openiddict/src/Audacia.Auth.OpenIddict.Seeding.EntityFrameworkCore/Audacia.Auth.OpenIddict.Seeding.EntityFrameworkCore.csproj

```diff
@@ -30,7 +30,7 @@
 			<IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
 		</PackageReference>
 		<PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="6.0.1" />
```

Comment:
This EF reference probably needs updating to EF 8

Reply:
Agreed over teams that this would require a .NET8 upgrade, which is beyond the scope of this PR